src/util/newconfig/ConfigFileJson.cpp

```diff
@@ -69,14 +74,17 @@ extractJsonValue(boost::json::value const& jsonValue)
     if (jsonValue.is_double()) {
         return jsonValue.as_double();
     }
+    if (jsonValue.is_null()) {
+        return NullType{};
+    }
     ASSERT(false, "Json is not of type int, uint, string, bool or double");
     std::unreachable();
```

Comment:
Not for this Pr, do we need `unreachable` here?

Reply:
Yeah, because this method returns something. Without `unreachable` compiler is not happy that we don't return anything.